src/runtime/cache.ts

```diff
@@ -254,7 +254,7 @@ export function defineCachedEventHandler<
       return true;
     },
     group: opts.group || "nitro/handlers",
-    integrity: [opts.integrity, handler],
+    integrity: hash([opts.integrity, handler]),
```

Comment:
(haven't locally checked the source code but we have to do something like this in order to allow completely overriding integrity - independent of handler's source hash)

```suggestion
    integrity: opts.integrity hash(handler),
```

Reply:
Ok, I see!